server/controllers/request.js

```diff
@@ -81,3 +83,63 @@ exports.updateRequest = asyncHandler(async (req, res, next) => {
     }
   });
 });
+
+exports.payPetSitter = asyncHandler(async (req, res, next) => {
```

Comment:
I added it to the arguments just to be consistent with the rest of the controllers

Reply:
If we aren't using it then it can be removed, arguments aren't really a huge deal in terms of consistency with the overall project. Things like using const vs function is consistency, using async/await instead of .then, etc.